dds_web/templates/user/confirm2fa.html

```diff
@@ -0,0 +1,47 @@
+{% extends "bootstrap/base.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block content %}
+<title>2FA</title>
```

Comment:
Not a significant issue, but would it be better to write something that a user can understand right away? 
Perhaps, DDS (either in acronym or full name) Multi-factor Authentication
I find it useful to have a clear title name for a browser tab which makes it easier to find it among many.

Reply:
Yes, I keep too many tabs to ever be able to read the title...

Reply:
Fixed!